services/workflows-service/src/rule-engine/core/condition/constants.ts

```diff
@@ -0,0 +1,13 @@
+import { EQUALS, BETWEEN, LT, LTE, GT, GTE, LAST_YEAR } from './helpers';
+
+import { ConditionType } from './enums';
+
+export const conditionHelpers = {
+  [ConditionType.EQUALS]: EQUALS,
+  [ConditionType.BETWEEN]: BETWEEN,
+  [ConditionType.GT]: GT,
+  [ConditionType.GTE]: GTE,
+  [ConditionType.LT]: LT,
+  [ConditionType.LTE]: LTE,
+  [ConditionType.LAST_YEAR]: LAST_YEAR,
+} as const;
```

Comment:
I'm not sure why the enum is necessary here. You can just do this:

```suggestion
import type { ValueOf } from 'type-fest';

export const conditionHelpers = {
  EQUALS,
  BETWEEN,
  GT,
  GTE,
  LT,
  LTE,
  LAST_YEAR,
} as const;

export type ConditionHelper = ValueOf<typeof conditionHelpers>;
```

Reply:
Module '"type-fest"' has no exported member 'ValueOf'

> I'm not sure why the enum is necessary here. You can just do this:

```
Module '"type-fest"' has no exported member 'ValueOf'
```

Reply:
We use an old version of `type-fest`...

You can use this:
```ts
export type ValueOf<ObjectType, ValueType extends keyof ObjectType = keyof ObjectType> = ObjectType[ValueType];
```